daemon/openrazer_daemon/dbus_services/dbus_methods/mamba.py

```diff
@@ -348,3 +348,54 @@ def get_supported_poll_rates(self):
     self.logger.debug("DBus call get_supported_poll_rates")
 
     return self.POLL_RATES
+
+
+@endpoint('razer.device.misc', 'setHyperPollingLED', in_sig='y')
+def set_hyperpolling_wireless_dongle_indicator_led_mode(self, mode):
```

Comment:
Just to clarify, who is meant to use this method? Frontends? Or the daemon should switch this when it does certain things?

Reply:
This is purely a user preference setting, the daemon doesn't ever need to change or set it. The hardware dongle has a single LED light that is configurable by the user to be in 1 of 3 states. This is the same in Synapse, you choose what you want the LED to represent (Connection Status, Battery Status, Battery Warning).

echo 1 > /sys/module/razermouse/drivers/hid:razermouse/*/hyperpolling_wireless_dongle_indicator_led_mode
where 1 = Connection Status, 2 = Battery Status, 3 = Battery Warning

Reply:
So led mode meaning here that e.g. you tell the mouse to display the connection status and then it'll on its own make the led (idk) red and green depending on if it's connected? Would be good to have this documented somewhere, probably best in this method in the daemon.

Reply:
Essentially yes, although the LED is part of the dongle and you're telling the dongle what function its LED performs. When using this dongle it becomes what openrazer communicates with (acting like a mouse), and in turn the dongle communicates with the mouse on your behalf. I added some documentation to the method.